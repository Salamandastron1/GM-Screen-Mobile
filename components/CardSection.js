import React from 'react';
import { View } from 'react-native';
import PropTypes from 'prop-types';

const CardSection = ({ children }) => (
  <View style={styles.containerStyle}>
    {children}
  </View>
);

const styles = {
  containerStyle: {
    padding: 5,
    alignItems: 'center',
    backgroundColor: 'inherit',
    flexDirection: 'row',
  },
};

CardSection.propTypes = {
  children: PropTypes.shape({}).isRequired,
};

export default CardSection;
