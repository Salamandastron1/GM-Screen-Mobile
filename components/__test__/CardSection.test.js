/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import CardSection from '../CardSection';

it('should match the shapshot', () => {
  const tree = shallow(<CardSection />);
  expect(tree).toMatchSnapshot();
});