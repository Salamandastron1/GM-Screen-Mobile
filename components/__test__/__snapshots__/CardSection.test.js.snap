// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the shapshot 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "inherit",
      "flexDirection": "row",
      "padding": 5,
    }
  }
/>
`;
